core/src/test/java/com/opensymphony/xwork2/ognl/OgnlUtilTest.java

```diff
@@ -375,7 +381,7 @@ public void testSetPropertiesDate() {
         assertEquals(cal.getTime(), foo.getBirthday());
 
         //UK style test
-        cal = Calendar.getInstance();
+        cal = Calendar.getInstance(Locale.UK);
```

Comment:
Why?

Reply:
Hello @aleksandr-m .

I found during debugging within an IDE that the `OgnlUtilTest` test `testSetPropertiesDate` was failing (even though it worked when run in the Maven build - on Windows in both cases).  It might be a combination of OS/environment conditions for the default Java locale - so I explicitly set the locale values (based on the existing comments) so that the test worked consistently both inside and outside the Maven build.

Reply:
Ok.